lib/src/components/NxSubmitMask/NxSubmitMask.tsx

```diff
@@ -23,14 +25,19 @@ const NxSubmitMask: FunctionComponent<Props> =
           'nx-submit-mask--success': !!success
         }),
 
-        // the default Loading... message is provided by NxLoadingSpinner
-        spinnerMessage = success ? (successMessage || 'Success!') : message;
+        showMessage = success ? (successMessage || 'Success!') : message;
```

Comment:
This is still confusing.  There's no reason to conflate the `message` and `successMessage` into the same variable.

Reply:
> This is still confusing. There's no reason to conflate the `message` and `successMessage` into the same variable.

How is this? https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/d88bd9fe8a996785f7bf86a97feed31447f2250d

Reply:
> How is this? d88bd9f

Better but the variables names you've chosen seem suggestive of being booleans, when they are actually strings.  You don't really need the variables at all; you can move the `foo || 'bar'` expressions directly into the JSX.

Reply:
> > How is this? [d88bd9f](https://github.com/sonatype/sonatype-react-shared-components/commit/d88bd9fe8a996785f7bf86a97feed31447f2250d)
> 
> Better but the variables names you've chosen seem suggestive of being booleans, when they are actually strings. You don't really need the variables at all; you can move the `foo || 'bar'` expressions directly into the JSX.

Oh cool! I can: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/a1c4908b760082684f29f30f6f839b54d5f3f397